python/GafferArnold/ArnoldTextureBake.py

```diff
@@ -262,6 +262,9 @@ def __init__( self, name = "ArnoldTextureBake" ) :
 		self["tasks"] = Gaffer.IntPlug( "tasks", defaultValue = 1 )
 		self["cleanupIntermediateFiles"] = Gaffer.BoolPlug( "cleanupIntermediateFiles", defaultValue = True )
 
+		self["applyMedianFilter"] = Gaffer.BoolPlug( "applyMedianFilter", Gaffer.Plug.Direction.In, False )
```

Comment:
Not something to act on right now, but just a note that the dictionary assignment will set the name anyway, and the default direction is in, so this could be more succinctly written as `self["applyMedianFilter"] = Gaffer.BoolPlug( defaultValue = False )`. I suspect you know this anyway, and it's more a case of this being adapted from a not-very-pretty-serialisation of a box?